generate_sample_file.py

```diff
@@ -101,9 +101,23 @@ def get_random_post_town(self):
     def get_random_post_code(self):
         first_random_number = random.randint(1, 9)
         second_random_number = random.randint(1, 9)
-        return f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number} {second_random_number}' + \
+        return f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number} {second_random_number}' \
                f'{self.get_random_letter()}{self.get_random_letter()}'
 
+    def get_random_field_coordinator_id(self):
+        first_random_number = random.randint(1, 9)
+        return f'{self.get_random_letter()}{self.get_random_letter()}-{self.get_random_letter()}' \
+               f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number}-' \
+               f'{self.get_random_letter()}{self.get_random_letter()}'
+
+    def get_random_field_officer_id(self):
+        first_random_number = random.randint(1, 9)
+        second_random_number = random.randint(1, 9)
+        third_random_number = random.randint(1, 9)
+        return f'{self.get_random_letter()}{self.get_random_letter()}-{self.get_random_letter()}' \
+               f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number}-' \
+               f'{self.get_random_letter()}{self.get_random_letter()}-{second_random_number}{third_random_number}'
```

Comment:
second and third number could be combined in the range 11 to 99... but no need to change it for something so minor